test/org/sagebionetworks/bridge/play/controllers/BaseControllerTest.java

```diff
@@ -60,10 +62,13 @@
 @SuppressWarnings("unchecked")
 public class BaseControllerTest {
     private static final String DOMAIN = "ws-test.sagebridge.org";
+    private static final String HEALTH_CODE = "health-code";
+    private static final String IP_ADDRESS = "dummy IP address";
```

Comment:
Should this and the different IP address be in IP address format just to make sure periods and whatnot don't throw off your string parsing?